src/main/scala/firrtl/transforms/CheckCombLoops.scala

```diff
@@ -24,6 +24,22 @@ object CheckCombLoops {
 
 case object DontCheckCombLoopsAnnotation extends NoTargetAnnotation
 
+case class CombinationalPath(sink: ComponentName, sources: Seq[ComponentName]) extends Annotation {
+  override def update(renames: RenameMap): Seq[Annotation] = {
+    val newSources = sources.flatMap { s =>
+      renames.get(s) match {
+        case None => Seq(s)
+        case Some(seq) => seq.collect{case c: ComponentName => c}
+      }
```

Comment:
Is the collect necessary? `RenameMap.get` with a `ComponentName` returns an `Optino[Seq[ComponentName]]`: https://github.com/freechipsproject/firrtl/blob/396ee7ca63eb8a9e201dcdea965cbfc3e9d36783/src/main/scala/firrtl/Compiler.scala#L56

Reply:
If that's the case, then you could just do `renames.getOrElse(s, Seq(s))`